deployment/v3/mosip/kernel/README.md

```diff
@@ -7,6 +7,11 @@ Refer [Commons](https://docs.mosip.io/1.2.0/modules/commons).
 ```
 ./install.sh
 ```
+* During the execution of the `install.sh` script, a prompt will appear, requesting information regarding the presence of a public domain and a valid SSL certificate on the server.
```

Comment:
During the execution of the `install.sh` script, a prompt appears requesting information regarding the presence of a public domain and a valid SSL certificate on the server.